docs/DeveloperGuide.md

```diff
@@ -185,7 +182,7 @@ The `Model` component,
 
 <div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.<br>
 
-❗️ IMAGE NEEDS TO BE UPDATED
+❗️ IMAGE NEEDS TO BE UPDATED (**Remove caution if image is sufficient**)
```

Comment:
Do you want to add that ScheduledMeeting is made out of MeetingDate and MeetingTime?

Reply:
Sure, that sounds good. But I am not sure if that is giving too much details as, if I am not wrong, this diagram is catered to show the ` UniqueTaglist`. But we can include it if you want. 
If you want, I can include those in the `ModelClassDiagram` instead, as you are right, the relationship between `ScehduledMeeting`, `MeetingDate` and `MeetingTime` is not clearly shown in our diagrams.

Reply:
Yup, I think if it's captured in at least one, it would be good enough 👍🏽